src/components/EventItem/EventItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import eventShape from '../helpers/propz/eventShape';
+
+import './EventItem.scss';
+
+class EventItem extends React.Component {
+  static propTypes = {
+    event: eventShape.eventShape,
+  }
+
+  render() {
+    const { event } = this.props;
+    console.log(event);
```

Comment:
Remove console logs

Reply:
Alrighty!  I just did that and repushed to the same branch.